docs/source/sysadmin_guide/about_dynamic_consensus.rst

```diff
@@ -0,0 +1,247 @@
+.. _about-dynamic-consensus-label:
+
+***********************
+About Dynamic Consensus
+***********************
+
+Sawtooth :term:`dynamic consensus` lets you choose from a variety of consensus
+algorithms for the network, including PBFT, PoET, Raft, and Devmode.
+
+* For a new network, the administrator for the first node specifies the
+  consensus settings in the :term:`genesis block`. All administrators must
+  must install and start a consensus engine (the same version) and any
+  transaction processors required for consensus (such as PoET Validator
+  Registry).
+  See :doc:`/sysadmin_guide/setting_up_sawtooth_poet-sim` for this procedure.
+
+* To change consensus for a running network, administrators must coordinate the
+  changes for all nodes. Each administrator must install and start the new
+  consensus engine. Then one administrator submits a transaction to change the
+  on-chain consensus settings.
+
+  After the consensus has changed, each administrator can stop the previous
+  consensus engine and any unnecessary transaction processors
+  (such as PoET Validator Registry).
+
+The rest of this topic summarizes the settings and requirements for each
+available consensus type: :ref:`PBFT <pbft-consensus-label>`,
+:ref:`PoET <poet-consensus-label>`, :ref:`Raft <raft-consensus-label>`, and
+:ref:`Devmode <devmode-consensus-label>`.
+
+.. note::
+
+   **Default Consensus Settings:**
+   Sawtooth provides defaults for backward compatibility with previous versions
+   of Sawtooth (1.0 and earlier), when Devmode and PoET were the only available
+   consensus algorithms.
+
+   * If ``sawtooth.consensus.algorithm.version`` is not set, the default version
+     is ``0.1``.
+
+   * If ``sawtooth.consensus.algorithm.name`` is not set, Sawtooth checks for
+     the deprecated setting ``sawtooth.consensus.algorithm``, which was used in
+     previous releases to specify PoET consensus. If this setting exists,
```

Comment:
This isn't 100% accurate; if this setting exists, it'll use that as the name. If this setting isn't `poet`, Sawtooth won't use PoET.